hcloud/images/client.py

```diff
@@ -113,10 +191,36 @@ def get_all(self,
                 sort=None,            # type: Optional[List[str]]
                 ):
         # type: (...) -> List[BoundImage]
+        """Get all images
+
+        :param name: str (optional)
+               Can be used to filter images by their name.
```

Comment:
```suggestion
               Name of the image to filter by.
```

Reply:
We should use the same description like in the API Docs. And there it is "Can be used to filter images by their name".